src/e2e-app/button/button-e2e.ts

```diff
@@ -1,4 +1,5 @@
-import {Component} from '@angular/core';
+import { Component } from '@angular/core';
```

Comment:
Why on earth this is in the style guide??? This is just a formatting choice that each project should make on its own. As long as formatting is consistent it should be none of style guide business....